src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,11 +5,14 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private final int weight;
+
     public Bulldozer() {
+        weight = 30;
```

Comment:
you should leave the no-args constructor untouched, but create another one with the weight field and initialize it in the place you use it, not hardcoding the value here